packages/mobile/src/providers/TokenProvider.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { createContext, ReactElement, useMemo, useState, useEffect } from 'react';
+
+import sdkService from '../services/sdk';
+import tokenStorage from '../utils/tokenStorage.util';
+
+import { URL_TOKEN, URL_TOKEN_KEYS } from './TokenUrl';
+
+const { retrieveToken } = tokenStorage();
+
+type TokenContext = {
+  token: string | null;
+  setToken: (value: string | null) => void;
+  refreshToken: (() => Promise<string>) | null;
+};
+
+type TokenProviderProps = {
+  children: ReactElement;
+  validateToken: (token: string) => boolean;
+  fetch: any;
```

Comment:
Get some warning for `any`. Is `any` a most?
If so, can we disable the warning?

Reply:
updated the type to be more specific